src/lightkurve/io/tess.py

```diff
@@ -46,4 +46,48 @@ def read_tess_lightcurve(
     lc.meta["TARGETID"] = lc.meta.get("TICID")
     lc.meta["QUALITY_BITMASK"] = quality_bitmask
     lc.meta["QUALITY_MASK"] = quality_mask
+    lc.meta["EXPTIME"] = lc.meta.get("FRAMETIM") * lc.meta.get("NUM_FRM")
+    lc.meta["CADENCE_TYPE"] = get_tess_cadence_type(lc.meta["EXPTIME"])
     return TessLightCurve(data=lc)
+
+def get_tess_cadence_type(exptime):
+    """ Returns the TESS cadence type as a string based on the exposure time.
+
+    Alternatively, if a string is pass then this function checks for a valid 
+    cadence type and then returns it. It raises an exception if not valid.
+
+    The options are:
+    if   exptime < 60   : "fast"  (i.e. 20-second)
+    elif exptime < 300  : "short" (i.e. 2-minute)
+    else                : "long"  (i.e. 30 or 10 minute FFI)
```

Comment:
I wonder if we want to use the string `"ffi"` instead of `"long"`, because I often hear people say "TESS FFI light curves" and I've ~never heard "TESS long cadence light curves".  (But I might be wrong -- is "long" is already adopted somewhere?)

Reply:
I'm debating this too. I've heard both. The mission uses FFI but I've seen others say "long". I would prefer "ffi'. If you are good with that then I'll change it to 'ffi'

Reply:
I think "ffi" is the way to go!

Reply:
Terminology changed to 'ffi' and merged to branch.